docs/templates.md

```diff
@@ -236,6 +236,7 @@ We recommend source controlling your templates as you would other code.
 CI is as simple as running `coder templates update` with the appropriate
 credentials.
 
----
 
-Next: [Authentication & Secrets](templates/authentication.md) and [Workspaces](./workspaces.md)
+## Next Steps
```

Comment:
Why use Next Steps here but Up next in the README.md?

Reply:
@ammario the text can be interchanged; In most places, Next Steps was used, in a few spots Up next was used. not sure why there was a slight difference in terminology so i left it as is and had roman extend the regex to include both wordings and thought the docs authors would sort it out. 

I can change all the text to read Next Steps, just let me know :)

(sorry i missed this til now)